static/js/fenced_code.js

```diff
@@ -189,6 +189,23 @@ exports.process_fenced_code = function (content) {
     return output.join('\n');
 };
 
+exports.get_unused_fence = (content) => {
+    // we only return ``` fences, not ~~~.
+    const used = new Set();
+    content.split('\n').forEach(line => {
+        const m = line.match(fence_re);
```

Comment:
This shouldn't be a concern; the regex we're matching against checks for opening fences, and thus any case of indented closing fence would be covered because of the closing fence. I added a testcase to verify this behavior.

Reply:
Fixed.